src/auto-evo/simulation/food_source/HeterotrophicFoodSource.cs

```diff
@@ -38,15 +38,24 @@ public override float FitnessScore(Species species, SimulationCache simulationCa
             var predatorSpeed = microbeSpecies.BaseSpeed;
             predatorSpeed += simulationCache.GetEnergyBalanceForSpecies(microbeSpecies, patch).FinalBalance;
 
-            // It's great if you can engulf this prey, but only if you can catch it
+            // Only assign engulf score if one can actually engulf
             var engulfScore = 0.0f;
             if (microbeSpeciesHexSize / preyHexSize >
                 Constants.ENGULF_SIZE_RATIO_REQ && !microbeSpecies.MembraneType.CellWall)
             {
                 engulfScore = Constants.AUTO_EVO_ENGULF_PREDATION_SCORE;
-            }
 
-            engulfScore *= predatorSpeed > preySpeed ? 1.0f : Constants.AUTO_EVO_ENGULF_LUCKY_CATCH_PROBABILITY;
+                // The faster the better
+                engulfScore *= predatorSpeed / preySpeed;
```

Comment:
I question this division here. The prey speed will be the same for all perspective predators, so my assumption would be that the only effect here would be to incentivize engulfing-based predation for slower prey, and disincentivize it for faster prey, rather than give any further advantage to a faster predator of the same prey.

If that was the plan, then the comment is a little misleading (but props for adding the comment)

Reply:
You're right, in that `preySpeed` definitely incentizes predation for slower preys. I guess you clarified it much better.